Frontend: App Branding!

Shrinks99 opened the pull request:
Closes #424 

### Changes
- Adds Browsertrix logo to the nav bar!!
  - Will license the typeface (Konsole) as soon as this gets merged.
- Changes the theme colour to Webrecorder's turquoise brand colour
- Adjusts the usage of `primary` vs `blue` for elements where it shouldn't be the same
- Changes the _Cancel & Discard_ button of the cancel crawl dialogue to `danger` button variant
- Replaces the Replay icon with the new ReplayWebpage icon!
- Fixes the favicon SVG linking (wrong name, oops!)

### Screenshots!

<img width="1022" alt="Screenshot 2024-03-12 at 6 03 58 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/5672810/20107d8b-ec7f-458c-b7de-8f2e2bc2fe48">

<img width="1378" alt="Screenshot 2024-03-24 at 9 27 09 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/5672810/ba5ebbcc-9a4b-4134-98f5-fdbbc04b3b74">

Shrinks99 committed:
Adds Browsertrix logo to nav bar

Shrinks99 committed:
Update app primary color

- Fixes a few instances where blue is hardcoded and shouldn't be.
- Fixes a few instances where _primary_ is hardcoded and shouldn't be!

Shrinks99 committed:
Changes cancel crawl dialog to use danger button for action

This is a destructive action and should be red!

emma-sg commented on frontend/src/assets/brand/browsertrix-lockup-color.svg:
this looks quite a big larger and more complicated than it needs to be for the logo — wanna run it through some cleanup tools?

Shrinks99 replied:
Was run through SVGO, if I put it through again I can remove an extra .09k

Typefaces have a lot of geometry!

emma-sg replied:
This could still be cleaned up a bit — lemme do another pass in a few min

Shrinks99 committed:
Remove browsertrix-lockup-color.svg doctype

SuaYoo reviewed (approved):
Nice!! I fixed a few merge conflicts, looks good to go.

Out of scope, potential follow-up: since "Browsertrix" is now bold and darker, it does take a lot of prominence in the app, when maybe the org name should be more prominent? We don't currently show the org name in the app bar if a user only belongs to one org, maybe we could display it as  `[Browsertrix logo mark] | [Org name]`.

Shrinks99 committed:
Merge branch 'main' into frontend-browsertrix-branding

Shrinks99 committed:
Fixes favicon path

At some point the name got mixed up, this fixes it :sweat_smile:

Shrinks99 committed:
Replaces replay icon with replaywebpage icon

Shrinks99 committed:
Update main logotype

Spacing update, suggestion applied from Emma

Shrinks99 committed:
Re-adds Browsertrix H1 to readme

Shrinks99 committed:
Replaces H1 by putting the image inside the H1

Apparently this is fine and you can do it as long as the image has an alt tag (which this does!)

Shrinks99 committed:
Remove "logo" from alt text

Shrinks99 committed:
Merge branch 'main' into frontend-browsertrix-branding

Shrinks99 committed:
Merge branch 'main' into frontend-browsertrix-branding

Shrinks99 committed:
Merge branch 'main' into frontend-browsertrix-branding

Shrinks99 committed:
Merge branch 'main' into frontend-browsertrix-branding

Shrinks99 committed:
Update replay tab icon in QA review

Shrinks99 committed:
Merge branch 'main' into frontend-browsertrix-branding

SuaYoo commented:
It looks like the logomark in the image isn't vertically centered. There's more space beneath than above.
<img width="212" alt="Screenshot 2024-04-17 at 7 29 40 PM" src="https://github.com/webrecorder/browsertrix-cloud/assets/4672952/cf15b4d3-3559-4077-9597-915623414fff">

Shrinks99 committed:
Fixes  brand lockup centering

Shrinks99 committed:
Class cleanup and redundant div removal

SuaYoo committed:
Full width app bar and main nav (#1693)

### Changes

App bar and org top-level navigation is full-width. Also, adds
background color to app bar to differentiate it from the org nav.

Shrinks99 committed:
Fills microscope icon

Shrinks99 committed:
Delete btrix-cloud.svg

Shrinks99 committed:
Center dynamic logo
